moped-database/metadata/tables.yaml

```diff
@@ -1,16 +1,16 @@
 - table:
```

Comment:
TODO: Figure out why this file re-scrambled? It should contain changes, but I was under the impression we shouldn't expect to see the `tables.yaml` jumble anymore.

Reply:
John's recent PR also had a scrambled tables.yaml :(

Reply:
@johnclary It's jumbled again. I think it's the result of maybe this PR being old and maybe created prior to your work to get the `tables.yaml` file ironed out and "in order." Maybe I can do what you did somehow on this branch to get it back in-line with `main`? I'm curious to what you think is smart. I don't want to regress the ordered nature of this file.

Reply:
i'm starting to wonder if hasura is just capricious and we'll be chasing the ordered life nature for a long time. i did find [this issue](https://github.com/hasura/graphql-engine/issues/6348) which confirms that Hasura does intend to be consistent with metadata sorting but also me have flaws.